lib/logstash/config/cpu_core_strategy.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/config/defaults"
+
+module LogStash module Config module CpuCoreStrategy
```

Comment:
hah, I think we discussed it a few weeks ago. I don't remember where (email?). I think the goal here was a hybrid between the brevity of `LogStash::Config::CpuCoreStrategy` and the scoping properties of `module Logstash\n module Config\n  module CpuCoreStrategy`